docs/_Research-Data-Management/26-licenses.md

```diff
@@ -51,6 +51,22 @@ There are different versions of CC that consist of the core license with further
 ## Usage
 Creators can declare a works license by selecting it during upload to a website/database/repository or attaching a line stating the title, author and license to the published work.
 
+Here we look at some examples of license usage, found on [Zenodo](https://zenodo.org/).
+
+### [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) example usage
+We are going to be looking at two examples here: 
+1. [Metagenome-assembled genomes (MAGs), colorectal cancer (CRC)](https://zenodo.org/records/7008911) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7008911.svg)](https://doi.org/10.5281/zenodo.7008911)
+2. [Metagenome assemblies and metagenome-assembled genomes from the Daphnia magna microbiota](https://zenodo.org/records/4435010) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4435010.svg)](https://doi.org/10.5281/zenodo.4435010)
+
+Both datasets have been submitted to Zenodo under the CC BY 4.0 license. That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the reuser must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under deed license.
```

Comment:
```suggestion
Both datasets have been submitted to Zenodo under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/) . That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the reuser must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under deed license.
```

I'm not sure reuser is a term, care to switch to user?

Reply:
```suggestion
Both datasets have been submitted to Zenodo under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/). That means that the data that has been deposited is free to share and free to redistribute (including commercially) in any format or medium. It also enables the data user to build upon or transform it for any purpose (including commercial use). However, the license requires the data user to give appropriate credit to the submitter/data generator. In addition, the user must also provide a link to the license and disclose any changes made when licensing their work when derived from work already under license.
```